src/components/Player/Player.jsx

```diff
@@ -38,6 +41,8 @@ const getPlayerSubroute = (info, playerId) => {
       return <ProsPage playerId={playerId} />;
     case 'rankings':
       return <RankingsPage playerId={playerId} />;
+    case 'histograms':
+      return <HistogramsPage playerId={playerId} histogramName={subInfo} />;
```

Comment:
I think you might have to connect it to the store to get it.  If you don't want to do that, passing here is fine.